Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -305,28 +300,91 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         dependencyMapper: DependencyMapper,
         fileSystem: FileSystem,
         observabilityScope: ObservabilityScope,
-        delegateQueue: DispatchQueue,
-        callbackQueue: DispatchQueue
+        delegateQueue: DispatchQueue
```

Comment:
Do we need the delegateQueue here anymore? there is no way we are FIFO when in swift concurrency so I am I am not sure that forcing a hop to the delegate queue is meaningful, but I haven't thought this all the way through